subliminal/core.py

```diff
@@ -357,6 +357,48 @@ def search_external_subtitles(path, directory=None):
     return subtitles
 
 
+def raise_for_invalid_file(path):
+    """Checks if is a valid file."""
+    # check for non-existing path
+    if not os.path.exists(path):
+        raise ValueError('Path does not exist')
+
+    # check that this is a file and not a directory
+    if not os.path.isfile(path):
+        raise ValueError('%r is not a file' % path)
```

Comment:
You shouldn't ever hit this exception unless you explicitly call `scan_video` on a directory. If you do that, don't expect the `scan_video` function to work properly. We are all consenting adults here